src/seedu/addressbook/commands/CreateCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.commands;
+
+
+import java.io.*;
+
+/**
+ * Adds a person to the address book.
+ */
+public class CreateCommand extends Command {
+
+    public static final String COMMAND_WORD = "create";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates a new user account. "
+            + "Parameters: USERNAME [p]p/PASSWORD \n"
+            + "Example: " + COMMAND_WORD
+            + " user123 p/password123 ";
+
+    public CreateCommand(String username, String password) {
+//        System.out.println("Manage to call \"create\" command");
+        System.out.println("New User Created: Username: " + username + " Password: " + password);
+
+        try { //TODO: extract this from CreateCommand
+            BufferedWriter writer = new BufferedWriter(new FileWriter("users.txt", true));
+
+            writer.write(username + ":" + password + "\n");
+            writer.newLine();
+            writer.close();
+        } catch (IOException e){
+            e.printStackTrace();
+        }
+    }
+
+    public static final String MESSAGE_ACCOUNT_CREATION_SUCCESSFUL = "Successfully created account!"; // Temporary message for testing
```

Comment:
should it be "your account is created Successfully!"?

Reply:
constants should be declared at the top of the class. I think the phrasing is ok but the other suggestions are good too.